src/Network/Network/help/Set-AzFirewall.md

```diff
@@ -165,11 +167,22 @@ $Hub = Get-AzVirtualHub -ResourceGroupName "testRG" -Name "westushub"
 $firewall.Allocate($Hub.Id)
 $firewall | Set-AzFirewall
 ```
+
 This example retrieves a Hub Firewall, deallocates the hub firewall, and saves it. The Deallocate command removes the reference 
 to the virtual hub but preserves the firewall's configuration. For changes to be reflected in cloud, Set-AzFirewall must be called.
 The Allocate method assigns the virtual hub reference to the firewall. Again, for changes to be reflected in cloud,
 Set-AzFirewall must be called.
 
+### 13:	Identify Top Fat Flows on Azure Firewall
+```powershell
+$azFw = Get-AzFirewall -Name "AzureFirewall" -ResourceGroupName "rg"
+$azFw.IdentifyTopFatFlows = $true
+
+$azFw | Set-AzFirewall
+```
+
```

Comment:
Could you provide sample output with following format
\```output
{output}
\```
If this example has no output, please fill it with `None`.

Reply:
@BethanyZhou Added the output and pushed.